client/packages/system/src/Patient/CreatePatientModal/FetchPatientModal.tsx

```diff
@@ -18,6 +19,7 @@ import {
   Typography,
 } from '@common/components';
 import { useFetchPatient } from './useFetchPatient';
+import { usePatientStore } from 'packages/programs/src';
```

Comment:
Ah and also fix this, needs to be `from '@openmsupply-client/programs'` for tests to run!